kythe/rust/indexer/src/indexer/analyzers.rs

```diff
@@ -94,14 +97,19 @@ impl<'a> UnitAnalyzer<'a> {
         // later to emit nodes and create a HashMap between a file path and its digest
         let mut file_vnames = HashMap::new();
         let mut file_digests = HashMap::new();
+        let mut generated_file_names = HashMap::new();
         for required_input in unit.get_required_input() {
             let analysis_vname = required_input.get_v_name();
             let storage_vname: VName = analysis_to_storage_vname(analysis_vname);
             let info = required_input.get_info();
-            let path = info.get_path().to_owned();
+            let source_path = info.get_path().to_owned();
+            let processed_path = storage_vname.get_path().to_owned();
+            if source_path != processed_path {
+                generated_file_names.insert(processed_path, source_path.clone());
```

Comment:
Based on the lookup in handle_files I think this is backwards?

Reply:
I don't believe so. This worked as intended on my test case

Reply:
Please include a rust_indexer_test of this behavior.